ci-operator/step-registry/openshift-qe/installer/bm/day2/prega-operators/openshift-qe-installer-bm-day2-prega-operators-commands.sh

```
@@ -13,17 +13,39 @@ bastion=$(cat ${CLUSTER_PROFILE_DIR}/address)
+
+  validate_yaml() {
+    local f="$1"
+    [[ -s "$f" ]] || return 1
+    if command -v yq &>/dev/null; then
+      yq eval 'has("kind") and has("apiVersion")' "$f" 2>/dev/null | grep -q '^true$'
+    else
+      grep -qE '^(apiVersion|kind):' "$f"
+    fi
+  }
+
+  if ! validate_yaml /tmp/idms_${OCP_VERSION}.yaml; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)